lib/tasks/multitenant_db_merge.rake

```diff
@@ -0,0 +1,350 @@
+             'created_at' => x['created_at'].asctime.in_time_zone("America/New_York"),
+             'updated_at' => x['updated_at'].asctime.in_time_zone("America/New_York"),
+             'line_id' => @line_mappings[x['line_id']],
+             'patient_id' => @patient_mappings[x['patient_id']],
+             'user_id' => @user_mappings[x['user_id']]
+           })
+    res['patient_id'].nil? ? nil : res
+  }
+  @call_list_entry_mappings = easy_mass_insert CallListEntry, 'call_list_entries', cle_map, false
+
+  # Port events
+  event_map = -> (x) {
+    res = x.except('id', 'created_at', 'updated_at', 'fund_id', 'line_id', 'patient_id')
+           .merge({
+             'fund_id' => @fund_id,
+             'created_at' => x['created_at'].asctime.in_time_zone("America/New_York"),
+             'updated_at' => x['updated_at'].asctime.in_time_zone("America/New_York"),
+             'line_id' => @line_mappings[x['line_id']],
+             'patient_id' => @patient_mappings[x['patient_id'].to_i],
+           })
+    res['patient_id'].nil? ? nil : res
+  }
+  @event_mappings = easy_mass_insert Event, 'events', event_map, false
+
+  # Versions, maybe a little more complicated...
+  # Essentially, versions have a lot of fields we have to kick over, both at the top level (item_id, whodunnit)
+  # and nested (object, object_changes).
+  # This is essentially a more complicated version of the lambdas above, except it changes anything that's known to be an id
+  # to the new value that's just been inserted into db. End effect is that all the ids stowed in versions change to the new appropriate value.
+  @fkey_mappings = {
+    'fund_id' => {@fund_id => @fund_id},
+    'clinic_id' => @clinic_mappings,
+    'pledge_generated_by_id' => @user_mappings,
+    'pledge_sent_by_id' => @user_mappings,
+    'last_edited_by_id' => @user_mappings,
+    'user_id' => @user_mappings,
+    'line_id' => @line_mappings,
+    'patient_id' => @patient_mappings,
+    'can_call_id' => @patient_mappings,
+    'can_pledge_id' => @patient_mappings,
+    'can_fulfill_id' => @patient_mappings,
+    'can_support_id' => @patient_mappings
+  }
+  @item_type_mappings = {
+    'ArchivedPatient' => @archived_patient_mappings,
+    'Call' => @call_mappings,
+    'CallListEntry' => @call_list_entry_mappings,
+    'Clinic' => @clinic_mappings,
+    'Config' => @config_mappings,
+    'ExternalPledge' => @external_pledge_mappings,
+    'Event' => @event_mappings,
+    'Fulfillment' => @fulfillment_mappings,
+    'Line' => @line_mappings,
+    'Note' => @note_mappings,
+    'Patient' => @patient_mappings,
+    'User' => @user_mappings,
+    'PracticalSupport' => @practical_support_mappings,
+  }
+
+  def transform_obj(obj, item_type, value_will_be_array)
+    obj.each_pair do |k, v|
+      obj['id'] = @item_type_mappings[item_type][v] if k == 'id'
+      if @fkey_mappings.keys.include? k || (k.start_with?('can_') && k.end_with?('_id'))
+        keyset = if k == 'can_call_id'
```

Comment:
```suggestion
        polymorphic_aware_mappings = if k == 'can_call_id'
```

Reply:
hm, this variable doesn't seem to actually get used (probably an orphan of debugging) - I'll just take it out.

Reply:
It was used! You accepted a suggested change from me that did the sibling rename on line 317 that did the rename as well as add a comment